classifiers/reference_quality/special_character_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,27 @@
+```python
+import unicodedata
+from typing import Optional, List, Tuple
+
+ATTRIBUTE: str = "text" # only text attributes
+ALLOWED_RANGES: List[int] = None # list of integers that represent Unicode code points
+LABEL: str = "has_special_character"
```

Comment:
first mandatory then optional otherwise it looks a bit weird in the integrator imo

```suggestion
ATTRIBUTE: str = "text" # only text attributes
LABEL: str = "has_special_character"
ALLOWED_RANGES: List[int] = None # list of integers that represent Unicode code points
```